packages/runtime/src/types/transport/IdentityDeletionProcessDTO.ts

```diff
@@ -0,0 +1,15 @@
+import { IdentityDeletionProcessStatus } from "@nmshd/transport";
+
+export interface IdentityDeletionProcessDTO {
+    id: string;
+    createdAt?: string;
+    createdByDevice?: string;
+    approvedAt?: string;
+    approvedByDevice?: string;
+    gracePeriodEndsAt?: string;
+    cancelledAt?: string;
+    cancelledByDevice?: string;
+    rejectedAt?: string;
+    rejectedByDevice?: string;
+    status: IdentityDeletionProcessStatus;
```

Comment:
I would change the order of the fields, so that `status` is at the same place like in the IdentityDeletionProcess.